packages/itwinui-variables/src/themes/dark-hc.scss

```diff
@@ -34,12 +34,12 @@ $bg-positive-1: hsl(118 81% 21%);
 $bg-positive-2: hsl(118 76% 28%);
 
 $bg-warning-0: hsl(60 1% 26%);
-$bg-warning-1: hsl(33 94% 25%);
-$bg-warning-2: hsl(33 90% 33%);
+$bg-warning-1: hsl(41.86 100% 21%);
```

Comment:
Should we round these hue values?

(images will likely change as a result)

Reply:
Rounded the values… but surprisingly image tests didn't change?